README.md

```diff
@@ -144,11 +141,15 @@ retrieved from somewhere using application-specific code and then passed
 to the `read` method.
 
 ```js
-const url = '/my-account/items/123';
-const capability = await getCapabilityFromDatabase({url}); // defined by your code
+const url = 'https://zcap.example/my-account/items/123';
+// defined by your code
+const capability = await getCapabilityFromDatabase({url, /* other */});
```

Comment:
It's not clear what "other" is supposed to be... happens multiple times in this PR (and examples).

Reply:
Well, it's "defined by your code".